src/platforms/pumpfun/event_parser.py

```
@@ -0,0 +1,485 @@
+        """
+        return [self._create_instruction_discriminator_bytes]
+    
+    def get_event_discriminators(self) -> list[bytes]:
+        """Get event discriminators for token creation.
+        
+        Returns:
+            List of event discriminator bytes to match
+        """
+        return [self._create_event_discriminator_bytes]
+    
+    def parse_token_creation_from_block(self, block_data: dict) -> TokenInfo | None:
```

Ruff (0.12.2)

`parse_token_creation_from_block` is too complex (24 > 10)

(C901)